tsl/test/sql/dist_hypertable.sql.in

```diff
@@ -965,8 +965,10 @@ SELECT detach_tablespace('tablespace2', 'disttable2');
 SELECT * FROM show_tablespaces('disttable2');
 DROP TABLE disttable2;
 
-DROP TABLESPACE tablespace1;
-DROP TABLESPACE tablespace2;
+SET client_min_messages = ERROR;
```

Comment:
this change seems not required

Reply:
Unfortunately, I think it needs to be there. Most of the tablespace tests do not drop the tablespace last in the test, so if the tablespace does *not* exist prior to this test being executed, the test will fail with an error. It is just pure luck that it executes after a test that actually create those tablespaces.